src/API/Management.php

```diff
@@ -180,6 +133,55 @@ public function __call(
      */
     public function getHttpClient(): HttpClient
     {
+        if ($this->httpClient === null) {
```

Comment:
Just CS: You could decide to exit early here to avoid indentation. 

```php
if ($this->httpClient !== null) {
  return $this->httpClient;
}
```

Reply:
Good call, that's much cleaner.